leanpkg/leanpkg/main.lean

```diff
@@ -43,13 +43,26 @@ d ← read_manifest,
 io.put_str_ln $ "configuring " ++ d.name ++ " " ++ d.version,
 assg ← solve_deps d,
 path_file_cnts ← mk_path_file <$> construct_path assg,
+io.put_str_ln path_file_cnts,
```

Comment:
This is just left-over debugging code, right?  If you actually want to print it, I think we should output it with some kind of description.